src/Microsoft.Health.Fhir.Liquid.Converter/Models/CodeSystemMapping.cs

```diff
@@ -6,7 +6,7 @@
 using System.Collections.Generic;
 using DotLiquid;
 
-namespace Microsoft.Health.Fhir.Liquid.Converter.Hl7v2.Models
+namespace Microsoft.Health.Fhir.Liquid.Converter.Models
```

Comment:
Can we change the name to `CodeMapping`, as both CodeSystem and ValueSet are mapped now?

Reply:
changed.

Reply:
Thanks!